modules/ui/src/main/resources/lucuma-css/lucuma-ui.scss

```diff
@@ -50,11 +50,11 @@
 }
 
 // For styling of buttons in primereact for different "compactness" and adjust other sizing
-button.p-component.p-button {
+.p-component.p-button {
```

Comment:
It turns out that the `.p-component` was enough to increase it's specificity over the primereact css which has the extra layer of `.dark-theme` or `.light-theme`.